bugzilla.js

```diff
@@ -182,38 +182,25 @@ function setup_queries() {
 
   var stale_range = make_search({
     chfield: "[Bug creation]",
-    chfieldfrom: "2016-04-25", // change to date of first nightly of next version at release
+    chfieldfrom: "2016-04-25",
     chfieldto: "Now",
-    f10: "OP",
-    f11: "cf_has_regression_range",
-    f12: "cf_has_regression_range",
-    f13: "CP",
-    f1: "OP",
-    f2: "keywords",
-    f3: "CP", 
-    f4: "OP",
-    f5: "cf_status_firefox49",
-    f6: "cf_status_firefox50",
-    f7: "cf_status_firefox51",
-    f8: "CP",
-    j10: "OR",
+    f1: "cf_status_firefox48",
+    f2: "cf_status_firefox49",
+    f3: "cf_status_firefox50",
+    f4: "cf_status_firefox51",
     j_top: "OR",
     keywords: "regression",
     keywords_type: "allwords",
-    o11: "equals",
-    o12: "equals",
-    o2: "substring",
-    o5: "nowords",
```

Comment:
I can't say I really understand the difference between `nowordssubstr` and `nowords` tbh but I would guess `nowords` is less likely to accidentally match another word.